bftengine/src/bcstatetransfer/RVBManager.hpp

```diff
@@ -159,6 +160,7 @@ class RVBManager {
   std::vector<RVBGroupId> stored_rvb_digests_group_ids_;
 
   // RVB data update during checkpointing / pruning
+  concord::util::ThreadPool digests_worker_pool_{"digests-thread-pool", 8};
```

Comment:
Offline pruning is already resource intensive so I am assuming it would be done in absence of daml/ethereum traffic. In such scenario I don't think 8 is big number. I agree that we need to find right set of threads for all different types of deployments having cpu range varying from 8 to 192 but its going to be a considerable effort.

Reply:
Here are the numbers for pruning 160K blocks on 7 node setup having 32 cpus.

N threads | Time spent in callback (in milliseconds)
0               | 2000 -2200 
2               | 860 - 990
4               | 450 - 750
8               | 320 - 450 

Time is represented in range above as every node took different time.
For example - 

![image](https://user-images.githubusercontent.com/1729669/218158964-c70a99be-9964-40d1-9ac7-9270c1034a78.png)